### PR TITLE
Change styles to fit with new theme

### PR DESCRIPTION
**Pull request by youngkidwarrior**

### CODE REVIEW

**Comment by youngkidwarrior** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/84?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

Current dependencies on/for this PR:
* `main`
  * **PR #80** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/80?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #81** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/81?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #83** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/83?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
        * **PR #84** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/84?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/84?utm_source=stack-comment).

File: packages/app/features/auth/sign-in-screen.tsx [hunk 1]
```diff
@@ -8,21 +8,35 @@ export const SignInScreen = () => {
       <Stack pos="relative" h="100svh" f={1} ai="center" jc="center">
         <View pos="absolute" top={0} left={0} mt="auto" mb="auto" zIndex={-1} w="100%" h="100%">
           <View mt="auto" mb="auto" w="100%" h="100%" mah="95svh">
-            <CornerTriangle corner="topLeft" pos="absolute" top={0} left={0} btw="$20" />
+            <CornerTriangle
+              corner="topLeft"
+              pos="absolute"
+              top={0}
+              left={0}
+              btw="273px"
```
Comment (0xBigBoss) [inline packages/app/features/auth/sign-in-screen.tsx:16]: pixels work on native?

**Review by 0xBigBoss — APPROVED** [review]

File: packages/app/features/auth/sign-in-screen.tsx (hunk 1 above)
Comment (youngkidwarrior) [reply packages/app/features/auth/sign-in-screen.tsx:16]: They do not. My bad good catch

**Review by 0xBigBoss — APPROVED** [review]

**Comment by youngkidwarrior** [comment]
### Merge activity

* **Feb 4, 9:22 PM EST**: @youngkidwarrior started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/84).

**Pull request merged**